2-mandatory/4-stocks.js

```diff
@@ -48,7 +59,15 @@ function getAveragePrices(closingPricesForAllStocks) {
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
 function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+  // TODO
+  let priceChangeOfStock = [];
+  for (let item of closingPricesForAllStocks) {
+    let priceChange = item[4] - item[0];
+    let formattedPriceChange = parseFloat(priceChange.toFixed(2));
+    priceChangeOfStock.push(formattedPriceChange);
+    priceChange = 0;
```

Comment:
no need to set priceChange as 0 here, as priceChange is being defined with each iteration